.github/workflows/ci.yml

```
@@ -17,18 +17,32 @@ on:
 jobs:
   # This workflow contains a single job called "build"
   build:
+    strategy:
+      matrix:
+        node: [10.x, 12.x, 14.x, 15.x]
```

Node 16 is now out so we should probably test against that instead of 15. Up to you, but we could also drop Node 10 as it's reached end of life?

```suggestion
        node: [10.x, 12.x, 14.x, 16.x]
```